lib/modules/dosomething/dosomething_campaign/includes/CampaignTransformer.php

```diff
@@ -62,7 +62,7 @@ public function index($parameters) {
   public function show($id) {
     try {
       $campaign = Campaign::get($id, 'full');
-      $campaign = services_resource_build_index_list($campaign, 'campaigns', 'id');
+      $campaign = array_pop(services_resource_build_index_list([$campaign], 'campaigns', 'id'));
```

Comment:
_note_: the `services_resource_build_index_list()` seems to require passing object within an array for it to function as before...